chroma-manager/tests/integration/shared_storage_configuration/test_command_alert.py

```diff
@@ -1,10 +1,10 @@
 
 from testconfig import config
-from django.utils.unittest import skipIf
+from django.utils.unittest import skip
 from tests.integration.core.chroma_integration_testcase import ChromaIntegrationTestCase
 
 
-@skipIf(not config.get('simulator'), "RealRemoteOperations can't fake out network failures")
+@skip("Needs setup on real hardware")
```

Comment:
Ditto.  We can simulate network failure on real systems, BTW.

Reply:
https://github.com/intel-hpdd/intel-manager-for-lustre/issues/519